### PR TITLE
add link to sdk support table

### PR DESCRIPTION
**Pull request by duglin**
ping @jskeet for quick review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,8 @@ events into CloudEvents, see
 ## SDKs
 
 In addition to the documentation mentioned above, there is also an
-[SDK proposal](cloudevents/SDK.md). A set of SDKs is also being developed:
+[SDK requirements](cloudevents/SDK.md) doc as well as a set of SDKs being
+developed:
 
 - [CSharp](https://github.com/cloudevents/sdk-csharp)
```
[the remark this answers is not in the document]
Comment (duglin) [reply README.md]: ok I made that change in a few more files for consistency

**Review by jskeet — APPROVED** [review]
I figure if you didn't want nitpicks, you wouldn't have mentioned me...

**Comment by duglin** [comment]
and thanks!

**Review by jskeet — COMMENTED** [review]
"(And in the translated versions.)" :)

**Comment by duglin** [comment]
got it

**Review by jskeet — APPROVED** [review]
Tempted to look really carefully to find a Columbo-like "just one more thing" - but nope, this looks good :)

**Comment by duglin** [comment]
Approved on the 4/4 call

**Pull request merged**